lib/redmine/menu_manager/menu_item.rb

```diff
@@ -88,6 +112,10 @@ def html_options(options = {})
   def add_condition(new_condition)
     raise ArgumentError, 'Condition needs to be callable' unless new_condition.respond_to?(:call)
     old_condition = @condition
-    @condition = -> (project) { old_condition.call(project) && new_condition.call(project) }
+    if old_condition.respond_to?(:call)
```

Comment:
I am curious why this change became necessary here. How can it be that this worked before?

Reply:
It did not work before, when there was initially no condition given for the element. Since the method is not used very often, and those elements who used it have a condition, we didn't notice it.